src/atomate2/vasp/jobs/lobster.py

```diff
@@ -0,0 +1,332 @@
+    outputs["additional_static_run_dir"] = additional_static_run_job_dir
+    outputs["additional_static_uuid"] = additional_static_run_uuid
+    outputs["lobster_uuids"] = []
+    outputs["lobster_dirs"] = []
+    outputs["lobster_task_documents"] = []
+
+    if lobstermaker is None:
+        lobstermaker = PureLobsterMaker()
+
+    for i, basis in enumerate(basis_dict):
+        lobsterjob = lobstermaker.make(
+            wavefunction_dir=wavefunction_dir, basis_dict=basis
+        )
+        lobsterjob.append_name(f"_run_{i}")
+        outputs["lobster_uuids"].append(lobsterjob.output.uuid)
+        outputs["lobster_dirs"].append(lobsterjob.output.dir_name)
+        outputs["lobster_task_documents"].append(lobsterjob.output)
+        jobs.append(lobsterjob)
+
+    flow = Flow(jobs, output=outputs)
+    return Response(replace=flow)
+
+
+@job
+def delete_lobster_wavecar(
+    dirs: list, dir_vasp: Path | str = None, dir_preconverge: Path | str = None
+):
+    """
+    Delete all WAVECARs.
+
+    Parameters
+    ----------
+    dirs: List[Path|str]
+        Path to directories of lobster jobs.
+    dir_vasp: Path or str
+        Path to directory of static VASP run.
+    dir_preconverge: Path or str
+        Path to directory of preconvergence run.
+    """
+    jobs = []
+    outputs: dict = {}
+    outputs["lobster_dir_name"] = []
+    outputs["add_static_dir_name"] = []
+    outputs["static_dir_name"] = []
+    dec_delete_files = job(delete_files)
+    for dir_name in dirs:
+        jobs.append(
+            dec_delete_files(
+                dir_name, include_files=["WAVECAR", "WAVECAR.gz"], allow_missing=True
+            )
+        )
+        outputs["lobster_dir_name"].append(dir_name)
+    if dir_preconverge is None and dir_vasp is not None:
+        dir_vasp_stat = strip_hostname(dir_vasp)
+        jobs.append(
+            dec_delete_files(
+                dir_vasp_stat,
+                include_files=["WAVECAR", "WAVECAR.gz"],
+                allow_missing=True,
+            )
+        )
+        outputs["static_dir_name"].append(dir_vasp_stat)
+    if dir_preconverge is not None and dir_vasp is not None:
+        dir_vasp_add_stat = strip_hostname(dir_preconverge)
+        dir_vasp_stat = strip_hostname(dir_vasp)
+        jobs.append(
+            dec_delete_files(
+                dir_vasp_stat,
+                include_files=["WAVECAR", "WAVECAR.gz"],
+                allow_missing=True,
+            )
+        )
+        jobs.append(
+            dec_delete_files(
+                dir_vasp_add_stat,
+                include_files=["WAVECAR", "WAVECAR.gz"],
+                allow_missing=True,
+            )
+        )
+        outputs["add_static_dir_name"].append(dir_vasp_add_stat)
+        outputs["static_dir_name"].append(dir_vasp_stat)
+
+    flow = Flow(jobs, output=outputs)
+    return Response(replace=flow)
+
+
+@job(output_schema=LobsterTaskDocument)
+def generate_database_entry(
```

Comment:
Is this needed? I can't see it used anywhere?

Reply:
Yeah, you are correct. I think this is from an earlier version of the workflow and I forgot to remove it. Sorry!